proposals/forms.py

```diff
@@ -642,9 +642,14 @@ def clean(self):
         return cleaned_data  # Sticking to Django conventions
 
 
-class StudyStartForm(forms.ModelForm):
+class StudyStartForm(SoftValidationMixin, ConditionalModelForm):
```

Comment:
I believe `SoftValidationMixin` is interacting strangely with the cleaning process, which is giving me an error. I'll have a PR into this branch soon with my proposed fix.